### DISCUSSION TITLE
How to disable wikiless?

### DISCUSSION THREAD

**Discussion by 621625** [searx/searx#3265] [General]
Hi there,

When I search for e. g. "hello wikipedia" I get the search term "hello" from a source called wikiless instead. 

Having tried wikiless, I have to say I kind of prefer the real wikipedia in some ways at least for now. How do I disable this feature that turns my wikipedia searches into the corresponding wikiless page instead?

**Comment by 621625** [comment]
Hi,

I would really like to be able to disable wikiless. Where does it come from? Why do I have to use it? It doesn't format correctly. And I didn't ask for it. I asked for wikipedia.

**Reply by kvch** [reply]
By default searx does not come with wikiless. This issue only impacts searxng, please start a discussion there.
